package.json

```diff
@@ -33,7 +33,5 @@
     "minify": "^6.0.1",
     "onchange": "^7.1.0"
   },
-  "dependencies": {
-    "reset-css": "^5.0.1"
-  }
+  "dependencies": {}
```

Comment:
do we need this?

Reply:
I just removed it. No, not necessary, I just left it for eventual future imports, but it's useless now.